src/Features/Core/Portable/Completion/CompletionRules.cs

```diff
@@ -180,7 +188,7 @@ public TextChange GetTextChange(CompletionItem selectedItem, char? ch = null, st
 
         protected virtual bool IsCommitCharacterCore(CompletionItem completionItem, char ch, string textTypedSoFar)
         {
-            return false;
+            return s_defaultCommitCharacters.Contains(ch);
```

Comment:
It isn't LINQ (see the usings). It's an extension method on Array the that calls IndexOf. I suppose bus making it a set might be quicker.